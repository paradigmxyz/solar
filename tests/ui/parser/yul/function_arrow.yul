{
    function f() -> {} //~ ERROR expected identifier
}
