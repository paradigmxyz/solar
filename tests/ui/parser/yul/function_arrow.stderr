error: expected identifier, found `{`
  --> $DIR/function_arrow.yul:2:21
   |
LL |     function f() -> {}
   |                     ^
   |

error: aborting due to 1 previous error

